Assets/Scripts/Explodable/ExplodableObject.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ExplodableObject : MonoBehaviour
+{
+
+    private float m_TerrainRevealRadius = 30.0f;
+    private static float m_TimeOfLastSFX = 0.0f;
+
+    private int timeDlay = 3;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        Destroy(this.gameObject, timeDlay);
+    }
+
+    void OnDestroy()
+    {
+        Collider[] colliders = Physics.OverlapSphere(transform.position, m_TerrainRevealRadius);
+
+        foreach (Collider c in colliders)
+        {
+            RevealableTerrain target = c.GetComponent<RevealableTerrain>();
```

Comment:
The code for revealing nearby objects and painting terrain looks very similar to the code in `RevealActor.cs`. Could we create a new instance of `RevealActor` and let that script handle it instead? That way we will not need to maintain two versions of terrain reveal.

Reply:
Actually this part is only there temporarily how big the explosion radius is :P
I'll remove this once I am done with the bomberman feature!